### PR TITLE
Add foreman_openscap 0.6.0 (DEB)

### PR DESCRIPTION
**Pull request by xprazak2**
* [x] Nightly
* [ ] 1.12
* [ ] 1.11
* [ ] 1.10

### CODE REVIEW

File: plugins/ruby-foreman-openscap/debian/control [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Source: ruby-foreman-openscap
+Section: ruby
+Priority: extra
+Maintainer: Foreman Openscap Team
+Build-Depends: debhelper, foreman (>= 1.12.0~rc1), ruby-foreman-deface (<<2.0.0)
```
Comment (mmoll) [inline plugins/ruby-foreman-openscap/debian/control]: If this works for 1.12, why should it not get cherry-picked to the 1.12 repos? :)

File: plugins/ruby-foreman-openscap/debian/copyright [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ruby-foreman-ansible
+Source: https://github.com/theforeman/foreman_ansible
+
+Files: *
+Copyright: 2015 Daniel Lobato Garcia <elobatocs@gmail.com>
+License: GPL-3.0+
```
Comment (mmoll) [inline plugins/ruby-foreman-openscap/debian/copyright]: this file needs the ansible -> openscap replacement

File: plugins/ruby-foreman-openscap/debian/postinst [hunk 3]
```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
```
Comment (mmoll) [inline plugins/ruby-foreman-openscap/debian/postinst]: depending on what this plugin is bringing (its own assets and/or API extension), postinst and rules need to get modified, see e.g. foreman_salt

File: plugins/ruby-foreman-openscap/foreman_ansible.rb [hunk 4]
```diff
@@ -0,0 +1 @@
+gem 'foreman_openscap', '0.6.0'
```
Comment (mmoll) [inline plugins/ruby-foreman-openscap/foreman_ansible.rb]: filename is still "ansible"

File: plugins/ruby-foreman-openscap/debian/control (hunk 1 above)
Comment (ares) [inline plugins/ruby-foreman-openscap/debian/control]: there were a lot of changes in 0.6, I'd prefer nightly for a while and then we can move to stable branch

**Comment by xprazak2** [comment]
@mmol thanks for your comments, I decided to close this after discussion with @ares because Debian does not seem to have packaged policies for Openscap.

**Pull request closed without merging**